src/App.tsx

```diff
@@ -1,61 +1,140 @@
+import React, { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+export const getUserById = (userId: number): User | null => {
+  const findUser = usersFromServer.find((user) => (user.id === userId));
+
+  return findUser || null;
+};
+
+export const getTodoId = (listTodos: Todo[]): number => {
+  const todosId = listTodos.map(todo => todo.id);
+
+  return Math.max(...todosId) + 1;
+};
```

Comment:
Do you need this exports when you used this functions in same file?
Move this functions to a separate file `helpers`.